lua/telescope/builtin.lua

```diff
@@ -950,4 +950,79 @@ builtin.keymaps = function(opts)
   }):find()
 end
 
+builtin.tags = function(opts)
+  opts = opts or {}
+
+  if not vim.loop.fs_open('tags', "r", 438) then
```

Comment:
`'tags'` file should be an option for `opts`. `opts.tag_filepath` or similar, so that you could run this for other tags if you wanted.